### ISSUE TITLE
`{fluid_content:prev_field_name`} outputs the wrong field name

### ISSUE THREAD

**Issue by shbchk** [ExpressionEngine/ExpressionEngine#4603]
I have this little hack: if I want to hide a block of a fluid field, I can place a special field (a toggle) called `hide_next_block` directly above this block.

After updating ExpressionEngine from version 7.4.11 to 7.5.4, it stopped working. The `{fluid_content:prev_field_name}` tag now outputs the wrong field name — not the one placed directly above it.

Here is an example of the template code:

```
{first_page_content}

{first_page_content:custom_button}
{if first_page_content:prev_field_name != "hide_next_block"}

{content}
<a href="{content:link}" role="button" class="custom-button">{content:text}</a>
{/content}

{/if}
{/first_page_content:custom_button}

{/first_page_content}
```

**Comment by bryannielsen** [comment]
This sounds like a duplicate of #4575 which we have a PR https://github.com/ExpressionEngine/ExpressionEngine/pull/4601 to fix if you want to try that solution before the next release

**Comment by bryannielsen** [comment]
@shbchk this should be resolved in 7.5.5
